BitwardenShared/UI/Vault/Vault/VaultList/VaultListItem.swift

```diff
@@ -111,6 +111,8 @@ extension VaultListItem {
                 Asset.Images.globe24
             case .secureNote:
                 Asset.Images.file24
+            case .sshKey:
+                Asset.Images.key16
```

Comment:
🎨 I think this should use an icon that's 24x24 e.g. `key24`. Otherwise, the stroke in the icon is too big if you resize the 16x16 icon to 24x24.

<img width="45" alt="Screenshot 2024-10-31 at 10 45 43 AM" src="https://github.com/user-attachments/assets/83096af2-4d1f-4f14-9a46-987d6383c28a">